prime-router/docs/proposals/0017-metadata-tables-cleanup.md

```diff
@@ -0,0 +1,441 @@
+    Maurice Reeves - qva8@cdc.gov
+    3 November 2022
+*/
+
+-- create a country table
+create table country
+(
+    country_id                  bigserial not null primary key
+    , country_name              varchar(512) not null
+    , country_abbreviation      varchar(50) not null
+);
+
+comment on table country is
+    'A table of countries for the messages we get. These should correspond to the ISO-3166 list';
+comment on column country.country_id is
+    'The primary key for the country table';
+comment on column country.country_name is
+    'The name of the country per ISO-3166, not the official state name value';
+comment on column country.country_abbreviation is
+    'This the alpha-3 code value from HL70399 or ISO-3166, for example USA not US';
+
+
+-- a table with state values
+create table state
+(
+    state_id                    bigserial not null primary key
+    , state_name                varchar(512) not null
+    , state_abbreviation        varchar(50) not null
+    , state_fips_code           varchar(50) null
+    , country_id                bigint not null references country(country_id)
+);
+
+comment on table state is
+    'A table of states for the messages we get. These should correspond to the USPS values for States, when dealing with US states.';
+comment on column state.state_id is
+    'The primary key for the state table';
+comment on column state.state_name
+    is 'The common name for the US state, territory, or province. For example, use Pennsylvania instead of The Commonwealth of Pennsylvania';
+comment on column state.state_abbreviation is
+    'The two-letter abbreviation used by the US Postal Service, or the correct abbreviation for Canadian addresses';
+comment on column state.state_fips_code is
+    'The FIPS numeric code used for the state. In the case of Canadian provinces, this would be null.';
+comment on column state.country_id is
+    'The foreign key for the country the state is in';
+
+
+-- a county table
+create table county
+(
+    county_id                   bigserial not null primary key
+    , county_name               varchar(512) not null
+    , county_fips_code          varchar(50) null
+    , state_id                  bigint not null references state(state_id)
+);
+
+comment on table county is
+    'A table for the county an address can be in.';
+comment on column county.county_id is
+    'The primary key for the county table';
+comment on column county.county_name is
+    'The name of the county';
+comment on column county.county_fips_code is
+    'The numeric code for the county per FIPS';
+comment on column county.state_id is
+    'The foreign key for the state a county is in. Per the Census Bureau no county can cross state lines';
+
+
+-- a facility could exist in multiple roles: ordering facility, testing facility,
+-- but in the future it could also be a servicing facility for vaccines or patient care
+-- at a hospital
+create table facility_role
```

Comment:
I would consider just folding this into the facility table. I'm not sure we get much advantage from having this as a separate table.